### PR TITLE
Removes now obsolete floorgrime path and icon

### PR DESCRIPTION
**Pull request by WJohn**
As of #39699 we now have effectively identical looking floorgrime icons coming in the form of decals, which means the original dirty looking plasteel floor icon "floorgrime" is obsolete since its effect can be applied to anything. As such, I have removed it from every map and the code.

HUGE thanks to antur for their map tool, babying me through it, and making hotfixes for me to work with our new map merger!

This touches quite a lot of maps, so I'm likely to get conflicts if this doesn't get merged after about a day.

### CODE REVIEW

**Comment by nicbn** [comment]
You replaced it with `obj/effect/decal/cleanable/dirt` instead of `/obj/effect/decal/cleanable/dirt` in some maps (not sure if this is an issue anyway).

**Comment by WJohn** [comment]
fuck

**Comment by WJohn** [comment]
That should fix it.

**Comment by nicbn** [comment]
Conflicts

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -20182,11 +20303,14 @@
 /obj/machinery/door/airlock{
 	name = "Locker Room"
 	},
-/obj/structure/disposalpipe/segment{
+<<<<<<< HEAD/turf/open/floor/plasteel,
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: no.

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -20182,11 +20303,14 @@
...
+/obj/effect/decal/cleanable/dirt,
+=======/obj/structure/disposalpipe/segment{
 	dir = 4
 	},
-/turf/open/floor/plasteel/floorgrime,
-/area/crew_quarters/locker)
+/turf/open/floor/plasteel,
+/obj/effect/decal/cleanable/dirt,
+>>>>>>> 7a52e44b0d19f4ee7e87fa92b699ed9a3e93e062/area/crew_quarters/locker)
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: .

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 2 above)
Comment (WJohn) [reply _maps/map_files/MetaStation/MetaStation.dmm]: heck

**Pull request merged**